### DISCUSSION TITLE
想問linechatbot連接

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#106] [Q&A] [answered]
想問一個有點蠢的問題
剛剛在做作業的時候，無論如何新增或是修改程式後執行
跑出來的東西都是一樣的
因此想請問一下連接作業的飯店管家
有沒有要先在電腦上做什麼設定才能以我的程式給飯店管家執行?

**Comment by wh20502** [comment, marked as answer]
我也有一樣的問題，建立新聊天機器人，關閉回復與開啟webhook，再建立ngrok以後設定secret & token還有webhook URL以後，然後還是沒辦法重現聊天機器人...

**Reply by wh20502** [reply]
我發現我的問題了，我設定完webhook後URL沒有勾選下面的use webhook，點選以後應該就可以加自己建立的機器人來測試老師給的程式了。希望有幫助到你。
